site/landing/layouts/index.html

```diff
@@ -228,9 +228,9 @@ <h3 class="h3 feature__headline">Vendor- and platform-agnostic</h3>
     <section id="integrate" class="cta">
       <h2 class="sr-only">Itegrate OptenTitan</h2>
```

Comment:
fix typos: "Integrate OpenTitan" (or change to "Contribute to OpenTitan")

Reply:
That's a non-rendering heading, I'll followup with a typo fix.

Reply:
Added as a commit to this PR.